src/resourcemgr/tpl/rootFolder.tpl

```diff
@@ -0,0 +1,12 @@
+<li class="root">
+	<a 
+		class="root-folder active"
+		href="#"
+		data-path="{{path}}" 
+		data-display="{{relPath}}"
+		data-total="{{#if total}}{{total}}{{else}}0{{/if}}"
+		data-children-limit="{{childrenLimit}}">
+		{{label}}
+	</a>
+	<ul></ul>
+</li>
```

Comment:
No new line at the end of the file

Reply:
@habemuscode you can add suggestions for a quick commit 😉
```suggestion
</li>

```